mgr/RootCling.pl

```diff
@@ -0,0 +1,429 @@
+#!/usr/bin/env perl
+#
+# Script to run rootcling $Lib $libDep $RootMap Cint_cxx list of h-files
```

Comment:
This file mgr/RootCling.pl is new. Where is it used? I see that mgr/RootCint.pl is used by mgr/Conscript-standard but no reference to mgr/RootCling.pl